README.adoc

```diff
@@ -163,6 +163,33 @@ to define and manage single instances across an application:
   @Inject @MyQualifier ManagedExecutor executor;
 ----
 
+[[context-propagation-defaults]]
+Context Propagation Defaults
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The MicroProfile Context Propagation specification does not define a default set of
+context types that must be propagated in the absence of explicit configuration.
+Implementations of MicroProfile Context Propagation are encouraged to default to
+propagating all available context types unless doing so would cause unexpected behavior.
+For consistency with the Jakarta/Java EE Concurrency programming model,
+transaction context may be cleared by default, rather than propagated by default,
+by implementations that are also Jakarta/Java EE application servers.
+
+// TODO: insert links to implementation defaults here, once available
+
+Applications can achieve portability across implementations with respect to propagated
+context types either by explicitly specifying the context propagation on the builder
+or by explicitly configuring the desired defaults via MicroProfile Config properties.
+For example,
+
+[source,java]
+----
+mp.context.ManagedExecutor.propagated=Application,CDI,Security
+mp.context.ManagedExecutor.cleared=Remaining
```

Comment:
So, in general this sort of configuration can be problematic for Quarkus users because RESTEasy and Servlet have their own context that users may be surprised to learn will not be in the CDI context.
Perhaps explicitely specifying cleared contexts is less dangerous?

Reply:
Explicitly specifying cleared contexts would be less disruptive in terms of continuing to use the implementation you are already on without trouble, but this seems like heading in the opposite direction to achieving portability because the RESTEasy context that the application wanted to assume was available has no guarantee of being available on another implementation.  With it showing in the propagated list, the application is at least stating its full requirements for an implementation that would need to be met by an implementation that is compatible.

That said, I can still switch to what you suggested.  I'll just need to remove the language about it being a way to achieve portability.  I'll add a second commit with that.